### DISCUSSION TITLE
Scrollen nach links / rechts keine Stations OTHs mehr da

### DISCUSSION THREAD

**Discussion by ds2k5** [wavelog/wavelog#458] [General]
Hallo,
wenn ich auf der Map nach links oder rechts Scrolle sind die Stations OTHs nicht mehr zu sehen.

Start:

![grafik](https://github.com/wavelog/wavelog/assets/11292286/e361e87b-c7bf-402c-a3d1-cb8b2cea15af)

Scroll links bzw. rechts

![grafik](https://github.com/wavelog/wavelog/assets/11292286/3b63e40a-9f8c-4f02-a2d4-a41c0e684d68)

Scrolle ich zurück sind die QTHs wieder da

Nutze Version: 1.6.0 von Wavelog

Firefox: 126.x
Chromium: 125.x

Ist das so gewollt ?

**Comment by HB9HIL** [comment]
Hallo @ds2k5 

das ist aufgrund der Leaflet Karte. Können wir aktuell nichts dran ändern, war bisher aber auch nie ein Problem :)
